chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/edition2015/RequiredAndRelatedCriteriaReviewer.java

```diff
@@ -168,20 +149,12 @@ private void checkE1CriterionHasRequiredDependencies(CertifiedProductSearchDetai
                 criterionService.get(Criteria2015.D_9))
                 .collect(Collectors.toList());
         CertificationCriterion e1 = criterionService.get(Criteria2015.E_1_OLD);
-        CertificationCriterion e1Cures = criterionService.get(Criteria2015.E_1_CURES);
 
         List<String> errors = validationUtils.checkSpecificCriterionForMissingComplementaryCriteriaErrors(
                 e1,
                 attestedCriteria,
                 requiredByE1Criteria);
         listing.getErrorMessages().addAll(errors);
-
-        errors = validationUtils.checkSpecificCriterionForMissingComplementaryCriteriaErrors(
```

Comment:
I also think this should be added back in. If it was removed correctly as part of OCD-3151, then one AQA test needs updated and I can point you to it.